tests/unit/mocks/mockPinia.ts

```diff
@@ -227,5 +227,5 @@ export const mockPinia = (appState: Partial<AppState> = {}, includeMetadata: boo
     },
   };
 
-  return createTestingPinia({ initialState, createSpy: vi.fn });
+  return createTestingPinia({ initialState, createSpy: vi.fn, stubActions });
```

Comment:
Another open PR, #47, adds the ability to stub actions from pinia in a bit more of a generic way, allowing the user of mockPinia to pass in a *dict* of options that is forwarded to createTestingPinia. Please could you copy the way this is done in the relevant commit: https://github.com/jameel-institute/daedalus-web-app/pull/47/commits/ae8adb3c1d060ecfe2017fa5c6417a5a507492ef

Reply:
Though actually (as I'm fixing elsewhere) I think the parameter
```ts
  testingOpts: TestingOptions = {},
```
should be
```ts
  testingOpts: Partial<TestingOptions> = {},
```